src/pages/VMView.vue

```diff
@@ -76,26 +95,31 @@ export default {
       this.loading = true
       var params = {
```

Comment:
Please do not use `var` unless it has to overcome its scope (not the case here).

Reply:
I replaced `var` with `let`